pkg/agent/params.go

```diff
@@ -104,15 +99,6 @@ func getParameters(config *datamodel.NodeBootstrappingConfiguration, generatorCo
 
 		if properties.WindowsProfile.HasCustomImage() {
 			addValue(parametersMap, "agentWindowsSourceUrl", properties.WindowsProfile.WindowsImageSourceURL)
```

Comment:
agentWindowsSourceUrl is also not needed, please remove it.

Reply:
also remove windowsAdminUsername and windowsAdminPassword

Reply:
Could we keep this so we can test the built VHD directly without publish it as a sig image or a pir image?